### ISSUE TITLE
进程死亡

### ISSUE THREAD

**Issue by wenkepei123456-lab** [Francis-Rings/StableAvatar#51]
你好，我遇到了一个进程被杀死的问题，但是我只是用1.3B那个模型，并且也是在A100 80G的卡上运行，有没有好人帮帮我！

就是在加载CLIP那个图像编码器的时候遇到了这个问题

# packages in environment at /opt/conda/envs/stablevatar:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
_openmp_mutex             5.1                       1_gnu  
absl-py                   2.3.1                    pypi_0    pypi
accelerate                1.10.1                   pypi_0    pypi
aiofiles                  24.1.0                   pypi_0    pypi
aiohappyeyeballs          2.6.1                    pypi_0    pypi
aiohttp                   3.12.15                  pypi_0    pypi
aiosignal                 1.4.0                    pypi_0    pypi
albucore                  0.0.24                   pypi_0    pypi
albumentations            2.0.8                    pypi_0    pypi
annotated-types           0.7.0                    pypi_0    pypi
antlr4-python3-runtime    4.9.3                    pypi_0    pypi
anyio                     4.10.0                   pypi_0    pypi
async-timeout             5.0.1                    pypi_0    pypi
attrs                     25.3.0                   pypi_0    pypi
audio-separator           0.17.2                   pypi_0    pypi
audioread                 3.0.1                    pypi_0    pypi
av                        15.1.0                   pypi_0    pypi
beartype                  0.18.5                   pypi_0    pypi
beautifulsoup4            4.13.5                   pypi_0    pypi
brotli                    1.1.0                    pypi_0    pypi
bzip2                     1.0.8                h5eee18b_6  
ca-certificates           2025.7.15            h06a4308_0  
certifi                   2025.8.3                 pypi_0    pypi
cffi                      1.17.1                   pypi_0    pypi
charset-normalizer        3.4.3                    pypi_0    pypi
click                     8.2.1                    pypi_0    pypi
coloredlogs               15.0.1                   pypi_0    pypi
cryptography              45.0.6                   pypi_0    pypi
cython                    3.1.3                    pypi_0    pypi
dashscope                 1.24.2                   pypi_0    pypi
datasets                  4.0.0                    pypi_0    pypi
decorator                 4.4.2                    pypi_0    pypi
decord                    0.6.0                    pypi_0    pypi
diffq                     0.2.4                    pypi_0    pypi
diffusers                 0.30.1                   pypi_0    pypi
dill                      0.3.8                    pypi_0    pypi
easydict                  1.13                     pypi_0    pypi
einops                    0.8.1                    pypi_0    pypi
exceptiongroup            1.3.0                    pypi_0    pypi
expat                     2.7.1                h6a678d5_0  
fastapi                   0.116.1                  pypi_0    pypi
ffmpy                     0.6.1                    pypi_0    pypi
filelock                  3.13.1                   pypi_0    pypi
flatbuffers               25.2.10                  pypi_0    pypi
frozenlist                1.7.0                    pypi_0    pypi
fsspec                    2024.6.1                 pypi_0    pypi
ftfy                      6.3.1                    pypi_0    pypi
gputil                    1.4.0                    pypi_0    pypi
gradio                    5.44.1                   pypi_0    pypi
gradio-client             1.12.1                   pypi_0    pypi
groovy                    0.1.2                    pypi_0    pypi
grpcio                    1.74.0                   pypi_0    pypi
h11                       0.16.0                   pypi_0    pypi
hf-xet                    1.1.9                    pypi_0    pypi
httpcore                  1.0.9                    pypi_0    pypi
httpx                     0.28.1                   pypi_0    pypi
huggingface-hub           0.34.4                   pypi_0    pypi
humanfriendly             10.0                     pypi_0    pypi
idna                      3.10                     pypi_0    pypi
imageio                   2.37.0                   pypi_0    pypi
imageio-ffmpeg            0.6.0                    pypi_0    pypi
importlib-metadata        8.7.0                    pypi_0    pypi
inquirerpy                0.3.4                    pypi_0    pypi
jinja2                    3.1.4                    pypi_0    pypi
joblib                    1.5.2                    pypi_0    pypi
julius                    0.2.7                    pypi_0    pypi
lazy-loader               0.4                      pypi_0    pypi
ld_impl_linux-64          2.40                 h12ee557_0  
libffi                    3.4.4                h6a678d5_1  
libgcc-ng                 11.2.0               h1234567_1  
libgomp                   11.2.0               h1234567_1  
librosa                   0.11.0                   pypi_0    pypi
libstdcxx-ng              11.2.0               h1234567_1  
libuuid                   1.41.5               h5eee18b_0  
libxcb                    1.17.0               h9b100fa_0  
llvmlite                  0.44.0                   pypi_0    pypi
markdown                  3.8.2                    pypi_0    pypi
markdown-it-py            4.0.0                    pypi_0    pypi
markupsafe                2.1.5                    pypi_0    pypi
mdurl                     0.1.2                    pypi_0    pypi
ml-collections            1.1.0                    pypi_0    pypi
ml-dtypes                 0.5.3                    pypi_0    pypi
moviepy                   1.0.3                    pypi_0    pypi
mpmath                    1.3.0                    pypi_0    pypi
msgpack                   1.1.1                    pypi_0    pypi
multidict                 6.6.4                    pypi_0    pypi
multiprocess              0.70.16                  pypi_0    pypi
ncurses                   6.5                  h7934f7d_0  
networkx                  3.3                      pypi_0    pypi
ninja                     1.13.0                   pypi_0    pypi
numba                     0.61.2                   pypi_0    pypi
numpy                     1.26.4                   pypi_0    pypi
nvidia-cublas-cu12        12.4.5.8                 pypi_0    pypi
nvidia-cuda-cupti-cu12    12.4.127                 pypi_0    pypi
nvidia-cuda-nvrtc-cu12    12.4.127                 pypi_0    pypi
nvidia-cuda-runtime-cu12  12.4.127                 pypi_0    pypi
nvidia-cudnn-cu12         9.1.0.70                 pypi_0    pypi
nvidia-cufft-cu12         11.2.1.3                 pypi_0    pypi
nvidia-curand-cu12        10.3.5.147               pypi_0    pypi
nvidia-cusolver-cu12      11.6.1.9                 pypi_0    pypi
nvidia-cusparse-cu12      12.3.1.170               pypi_0    pypi
nvidia-cusparselt-cu12    0.6.2                    pypi_0    pypi
nvidia-nccl-cu12          2.21.5                   pypi_0    pypi
nvidia-nvjitlink-cu12     12.4.127                 pypi_0    pypi
nvidia-nvtx-cu12          12.4.127                 pypi_0    pypi
omegaconf                 2.3.0                    pypi_0    pypi
onnx                      1.19.0                   pypi_0    pypi
onnx-weekly               1.20.0.dev20250901          pypi_0    pypi
onnx2torch                1.5.15                   pypi_0    pypi
onnx2torch-py313          1.6.0                    pypi_0    pypi
onnxruntime-gpu           1.22.0                   pypi_0    pypi
opencv-python             4.11.0.86                pypi_0    pypi
opencv-python-headless    4.11.0.86                pypi_0    pypi
openssl                   3.0.17               h5eee18b_0  
orjson                    3.11.3                   pypi_0    pypi
packaging                 25.0                     pypi_0    pypi
pandas                    2.3.2                    pypi_0    pypi
pfzy                      0.3.4                    pypi_0    pypi
pillow                    11.0.0                   pypi_0    pypi
pip                       25.1               pyhc872135_2  
platformdirs              4.4.0                    pypi_0    pypi
pooch                     1.8.2                    pypi_0    pypi
proglog                   0.1.12                   pypi_0    pypi
prompt-toolkit            3.0.52                   pypi_0    pypi
propcache                 0.3.2                    pypi_0    pypi
protobuf                  6.32.0                   pypi_0    pypi
psutil                    7.0.0                    pypi_0    pypi
pthread-stubs             0.3                  h0ce48e5_1  
pyarrow                   21.0.0                   pypi_0    pypi
pycparser                 2.22                     pypi_0    pypi
pydantic                  2.11.7                   pypi_0    pypi
pydantic-core             2.33.2                   pypi_0    pypi
pydub                     0.25.1                   pypi_0    pypi
pygments                  2.19.2                   pypi_0    pypi
python                    3.10.18              h1a3bd86_0  
python-dateutil           2.9.0.post0              pypi_0    pypi
python-multipart          0.0.20                   pypi_0    pypi
pytz                      2025.2                   pypi_0    pypi
pyyaml                    6.0.2                    pypi_0    pypi
readline                  8.3                  hc2a1206_0  
regex                     2025.8.29                pypi_0    pypi
requests                  2.32.5                   pypi_0    pypi
resampy                   0.4.3                    pypi_0    pypi
rich                      14.1.0                   pypi_0    pypi
rotary-embedding-torch    0.6.5                    pypi_0    pypi
ruff                      0.12.11                  pypi_0    pypi
safehttpx                 0.1.6                    pypi_0    pypi
safetensors               0.6.2                    pypi_0    pypi
samplerate                0.1.0                    pypi_0    pypi
scikit-image              0.25.2                   pypi_0    pypi
scikit-learn              1.7.1                    pypi_0    pypi
scipy                     1.15.3                   pypi_0    pypi
semantic-version          2.10.0                   pypi_0    pypi
sentencepiece             0.2.1                    pypi_0    pypi
setuptools                78.1.1          py310h06a4308_0  
shellingham               1.5.4                    pypi_0    pypi
simsimd                   6.5.1                    pypi_0    pypi
six                       1.17.0                   pypi_0    pypi
sniffio                   1.3.1                    pypi_0    pypi
soundfile                 0.13.1                   pypi_0    pypi
soupsieve                 2.8                      pypi_0    pypi
soxr                      0.5.0.post1              pypi_0    pypi
sqlite                    3.50.2               hb25bd0a_1  
starlette                 0.47.3                   pypi_0    pypi
stringzilla               3.12.6                   pypi_0    pypi
sympy                     1.13.1                   pypi_0    pypi
tensorboard               2.20.0                   pypi_0    pypi
tensorboard-data-server   0.7.2                    pypi_0    pypi
threadpoolctl             3.6.0                    pypi_0    pypi
tifffile                  2025.5.10                pypi_0    pypi
timm                      1.0.19                   pypi_0    pypi
tk                        8.6.15               h54e0aa7_0  
tokenizers                0.21.4                   pypi_0    pypi
tomesd                    0.1.3                    pypi_0    pypi
tomlkit                   0.13.3                   pypi_0    pypi
torch                     2.6.0+cu124              pypi_0    pypi
torchaudio                2.6.0+cu124              pypi_0    pypi
torchdiffeq               0.2.5                    pypi_0    pypi
torchsde                  0.2.6                    pypi_0    pypi
torchvision               0.21.0+cu124             pypi_0    pypi
tqdm                      4.67.1                   pypi_0    pypi
trampoline                0.1.2                    pypi_0    pypi
transformers              4.51.3                   pypi_0    pypi
triton                    3.2.0                    pypi_0    pypi
typer                     0.17.3                   pypi_0    pypi
typing-extensions         4.12.2                   pypi_0    pypi
typing-inspection         0.4.1                    pypi_0    pypi
tzdata                    2025.2                   pypi_0    pypi
urllib3                   2.5.0                    pypi_0    pypi
uvicorn                   0.35.0                   pypi_0    pypi
wcwidth                   0.2.13                   pypi_0    pypi
websocket-client          1.8.0                    pypi_0    pypi
websockets                15.0.1                   pypi_0    pypi
werkzeug                  3.1.3                    pypi_0    pypi
wheel                     0.45.1          py310h06a4308_0  
xorg-libx11               1.8.12               h9b100fa_1  
xorg-libxau               1.0.12               h9b100fa_0  
xorg-libxdmcp             1.1.5                h9b100fa_0  
xorg-xorgproto            2024.1               h5eee18b_1  
xxhash                    3.5.0                    pypi_0    pypi
xz                        5.6.4                h5eee18b_1  
yarl                      1.20.1                   pypi_0    pypi
zipp                      3.23.0                   pypi_0    pypi
zlib                      1.2.13               h5eee18b_1 

./inference.sh: 第 49 行： 16856 已杀死               CUDA_VISIBLE_DEVICES=1 python inference.py --config_path="deepspeed_config/wan2.1/wan_civitai.yaml" --pretrained_model_name_or_path=$MODEL_NAME --transformer_path="checkpoints/StableAvatar-1.3B/transformer3d-square.pt" --pretrained_wav2vec_path="checkpoints/wav2vec2-base-960h" --validation_reference_path="examples/case-1/reference.png" --validation_driven_audio_path="examples/case-1/audio.wav" --output_dir="output_infer" --validation_prompts="A middle-aged woman with short light brown hair, wearing pearl earrings and a blue blazer, is speaking passionately in front of a blurred background resembling a government building. Her mouth is open mid-phrase, her expression is engaged and energetic, and the lighting is bright and even, suggesting a television interview or live broadcast. The scene gives the impression she is singing with conviction and purpose." --seed=42 --ulysses_degree=1 --ring_degree=1 --motion_frame=25 --sample_steps=50 --width=512 --height=512 --overlap_window_length=5 --clip_sample_n_frames=81 --GPU_memory_mode="model_full_load" --sample_text_guide_scale=3.0 --sample_audio_guide_scale=5.0

**Comment by Francis-Rings** [comment]
Hi, please check if your memory is full. I was unable to reproduce this issue with my A100 80GB. Additionally, you could try creating a Docker container and running StableAvatar within it to see if the issue exists."
